test/test_dataloader.py

```diff
@@ -3477,6 +3478,9 @@ def _run_ind_worker_queue_test(self, batch_size, num_workers):
             if current_worker_idx == num_workers:
                 current_worker_idx = 0
 
+    @unittest.skipIf(
+        IS_WINDOWS or IS_MACOS, "https://github.com/pytorch/pytorch/issues/68643"
```

Comment:
maybe add more info in the skip message?
unrelated, i wonder if we want a flaky marker